### ISSUE TITLE
Something wrong with backtick positioning

### ISSUE THREAD

**Issue by stevejpurves** [curvenote/editor#61]
When I try to write some inline code using a backtick, the initial position of the backtick is wrong.

![TNpWY7LJvm](https://user-images.githubusercontent.com/1473646/136012283-617e3fe1-ecc8-4ec1-8df5-cb6bf821e6b4.gif)

**Comment by rowanc1** [comment]
Yeah. This is a font update issue. Super annoying. I think we need to change our font.
